### PR TITLE
Bump go to stable version

### PR DESCRIPTION
**Pull request by matthewfala**
Bumping go to stable version to avoid security issues.

Signed-off-by: Matthew Fala <falamatt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,9 +15,7 @@ all: release
 
 .PHONY: release
 release:
-	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
 	docker build -t amazon/aws-for-fluent-bit:latest -f Dockerfile .
-	docker build -t amazon/aws-for-fluent-bit:init-latest -f Dockerfile.init .
```
Comment (matthewfala) [inline Makefile:20]: for testing only. Will add these lines back.

**Review by zhangyuzhu13 — APPROVED** [review]

**Review by DrewZhang13 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (PettitWesley) [reply Makefile:20]: Please remove so I can approve

Comment (PettitWesley) [reply Makefile:20]: Please remove so I can approve

**Review by PettitWesley — CHANGES_REQUESTED** [review]

**Review by PettitWesley — APPROVED** [review]
Giving my approval so you are unblocked, but fix the thing noted please

**Review by DrewZhang13 — APPROVED** [review]

**Comment by matthewfala** [comment]
Screen Shot 2022-10-10 at 6 28 36 PM
Proof cloudwatch still works

**Comment by matthewfala** [comment]
Great, thank you drew. Adding release notes

**Pull request merged**